WalletWasabi.Fluent/ViewModels/CoinSelection/Core/Headers/AnonymityScoreHeaderViewModel.cs

```diff
@@ -0,0 +1,5 @@
+namespace WalletWasabi.Fluent.ViewModels.CoinSelection.Core.Headers;
+
+internal class AnonymityScoreHeaderViewModel : ViewModelBase
+{
+}
```

Comment:
I keep thinking that these empty VMs just for the sake of template selection make no sense.

We can manually specify `DataTemplate`s in XAML, like [here](https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Fluent/Views/Wallets/Send/TransactionPreviewView.axaml#L129)

Reply:
Notice how you still use a class to specify the `DataType`. If I remove the class, things won't work :)